fecgov/openFEC
# Remove archived_mur_names.csv

## Summary (required)

- Resolves #4606

_Delete archived_mur_names.csv from repo.  The data is in database now_

## How to test the changes locally

- Download feature branch
- archived_mur_names.csv will not be found in this folder: openFEC/data/
- run `pytest` and make sure all tests pass
